fitbenchmarking/HTML_templates/index_page.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html>
+<meta charset="UTF-8">
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
+<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Raleway">
+<style>
+body,h1,h2,h3,h4,h5 {ofnt-family: "Raleway", sans-serif}
+table {
+  border-collapse: collapse;
+}
+
+table, th, td {
+  border: 1px solid black;
+}
+</style>
+
+<html>
+<head>
+    <title>{{ group_name }} results</title>
+</head>
+<body>
+    <section class="row">
+        <header class="large-12 columns">
+            <h1>Fitbenchmarking results: {{ group_name }}</h1>
+            <hr>
+        </header>
+    </section>
+    FitBenchmarking is a tool for comparing different minimizers/fitting software based on their accuracy and runtimes.
```

Comment:
Maybe just different fitting software (i.e., remove minimizers)?

Reply:
Corrected

Reply:
This doesn't seem to have been updated - has it been pushed?